### PR TITLE
Implemented Reject message

### PR DESCRIPTION
**Pull request by smh**
Added (very simple) tests, all tests passed, code coverage still at 100% and no new lint issues added. Let me know if there is anything I should change.

### CODE REVIEW

File: lib/messages/commands/reject.js [hunk 1]
```diff
@@ -3,19 +3,65 @@
...
+  this.message = arg.message;
+  this.ccode = arg.ccode;
+  this.reason = arg.reason;
+  this.data = arg.data;
 }
 inherits(RejectMessage, Message);
 
-RejectMessage.prototype.setPayload = function() {};
+RejectMessage.CCODE = {
+  REJECT_MALFORMED:       0x01,
```
Comment (braydonf) [inline lib/messages/commands/reject.js]: minor nit: spacing should be similar with the rest of the code base (e.g. https://github.com/bitpay/bitcore/blob/master/lib/crypto/signature.js#L307)

File: test/messages/commands/index.js [hunk 2]
```diff
@@ -263,6 +264,48 @@ describe('Command Messages', function() {
...
+        data: new Buffer('12345678901234567890123456789012', 'hex')
+      });
+      var payload = message.getPayload();
+      message = messages.Reject();
+      message.setPayload(payload);
+      message.message.should.equal('tx');
+      message.ccode.should.equal(0x01);
+      message.reason.should.equal('transaction is malformed');
+      message.data.toString('hex').should.equal('12345678901234567890123456789012');
+      //var message = messages.Reject();
```
Comment (braydonf) [inline test/messages/commands/index.js]: Any reason these are left commented out?

File: lib/messages/commands/reject.js (hunk 1 above)
Comment (smh) [inline lib/messages/commands/reject.js]: My bad, should have checked the coding conventions before sending the pull request. Will fix as soon as I can, currently on the road from Spain back home to Norway.

- Stein

On August 12, 2015 at 19:10:40 GMT+2, Braydon Fuller <notifications@github.com> wrote:In lib/messages/commands/reject.js:  > } > inherits(RejectMessage, Message); > > -RejectMessage.prototype.setPayload = function() {}; > +RejectMessage.CCODE = { > + REJECT_MALFORMED: 0x01,   minor nit: spacing should be similar with the rest of the code base (e.g. https://github.com/bitpay/bitcore/blob/master/lib/crypto/signature.js#L307)  —Reply to this email directly or view it on GitHub.

File: test/messages/commands/index.js (hunk 2 above)
Comment (smh) [inline test/messages/commands/index.js]: Looks like I forgot to take it out, will remove

-  Stein

On August 12, 2015 at 19:11:40 GMT+2, Braydon Fuller <notifications@github.com> wrote:In test/messages/commands/index.js:  > + }); > + it('should write payload correctly', function() { > + var message = messages.Reject({ > + message: 'tx', > + ccode: 0x01, > + reason: 'transaction is malformed', > + data: new Buffer('12345678901234567890123456789012', 'hex') > + }); > + var payload = message.getPayload(); > + message = messages.Reject(); > + message.setPayload(payload); > + message.message.should.equal('tx'); > + message.ccode.should.equal(0x01); > + message.reason.should.equal('transaction is malformed'); > + message.data.toString('hex').should.equal('12345678901234567890123456789012'); > + //var message = messages.Reject();   Any reason these are left commented out?  —Reply to this email directly or view it on GitHub.

**Comment by smh** [comment]
Seems I had forgotten to get back to this pull request. Fixed spacing and removed commented out old code. Rebased and should apply cleanly.

**Comment by braydonf** [comment]
Great, thanks for updating!

**Comment by braydonf** [comment]
LGTM

**Comment by braydonf** [comment]
Released in v1.1.0

**Pull request merged**